doc/source/reference/random/multithreading.rst

```diff
@@ -61,48 +57,59 @@ seed will produce the same outputs.
             self.executor.shutdown(False)
 
 
+
 The multithreaded random number generator can be used to fill an array.
 The ``values`` attributes shows the zero-value before the fill and the
 random value after.
 
 .. code-block:: ipython
 
-    In [2]: mrng = MultithreadedRNG(10000000, seed=0)
-    ...: print(mrng.values[-1])
-    0.0
+    In [2]: mrng = MultithreadedRNG(10000000, seed=12345)
```

Comment:
@rkern I was wondering about `seed=...` best practices for documentation (not necessarily here), do you think this is fine, or should we write such documentation in the form of `seed=0x...`. I have probably asked this before, but...

Reply:
Good question. Avoiding `seed=0` is important, IMO. Thanks, @fzeiser! Grabbing a random uint64 to use for a seed in the examples might help encourage safer practices. That is, the example writer executes something like the following in their Python shell

```
>>> hex(np.random.default_rng().integers(1<<64, dtype=np.uint64))
'0xdf05c6f9894cea6b'
```

Then use that value in their example:

```
"""
Examples
--------
In [2]: mrng = MultithreadedRNG(10000000, seed=0xdf05c6f9894cea6b)
"""
```

But I'm not going to complain too much about using `12345`. That does help communicate "this is a dummy value", so I get why example writers use it. I'm not sure that seeing `seed=12345` in an example influences too many people to use `seed=12345` in their own code.

Reply:
Well, I am more thinking about pushing best practices without actually forcing too much on users. I.e. we could make examples generally have a comment:
`# seed found using: hex(np.random.default_rng().integers(1<<64, dtype=np.uint64))`
(or anything prettier/shorter if we can think of it)

Reply:
If you think it is good practice, I would be happy with adding:
```
def create_unique_seed():
    return np.random.default_rng().integers(1<<64, dtype=np.uint64)
    # or maybe: return np.random.SeedSequence().entropy 
```
just so that such a comment can be short.

Reply:
I think we talked about having such a function back in the old mega-thread but held off on it to get some practice with the new system and `SeedSequence` in particular. Let's think about it for 1.19.